Swift/BRCrypto/common/BRBlockChainDB.swift

```diff
@@ -339,7 +339,8 @@ public class BlockChainDB {
             acknowledgements: UInt64,
             index: UInt64,
             transactionId: String?,
-            blockchainId: String)
+            blockchainId: String,
+            metaData: Dictionary<String,String>?)
```

Comment:
Has this change been done in Java?

Reply:
Yes, in `announceGetTransfersItemGen()`.  There is is destructures into its keys+values and passed onto Crypto C.